### PR TITLE
Add individual `Grade`, `Judgement/{game mode}`, and `Score/{game mode}` articles

### PR DESCRIPTION
**Pull request by TicClick**
part of #6431. some translations were moved -- why throw out something that's more or less up to date?

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

to have everyone on the same page, here's what we have now:

<details>
<summary>Details</summary>

```
Beatmapping
└── Health
Game mode
├── osu! and other game modes
    ├── ## Grade
    ├── ## Accuracy
    ├── ## Scoring
    ├── ## Judgement
    ├── ## Combo
    └── ## Health mechanics
Gameplay
└── Accuracy
Glossary
└── Combo (score multiplier)
Score
├── osu!
│   ├── ## Hit values
│   └── ## Scoring
└── osu!catch
    └── ... (same headings)
```

</details>

here's what this PR does:

<details>
<summary>Details</summary>

```
Beatmapping
└── Health
Game mode
└── osu!
    ├── ## Grade            <- to be removed later
    ├── ## Accuracy         <- to be removed later
    ├── ## Scoring          <- to be removed later
    ├── ## Judgement        <- to be removed later
    ├── ## Combo            <- to be removed later
    └── ## Health mechanics <- to be removed later
Gameplay
├── Accuracy
├── Combo (score multiplier)
├── Grade
└── Judgement
    └── osu! and other game modes
        ├── ## Hit object values
        └── ## Judgement
Score
└── osu! and other game modes
    └── ## Scoring
```

</details>

### CODE REVIEW

**Review by Satsukel — CHANGES_REQUESTED** [review]
Hello, so it's kind of a coincidence that I'm suggesting changes for your PR but a friend mentioned me that the scoring system written on Osu!mania game play page is out of date.
To resume what my modifications are about: the "ModDivider" doesn't exist in Osu!mania because it suggests that by playing with DT (for example) we can get more than 1 million points which is wrong for that mode. 
Unless I misunderstood its purpose, I clearly think that we should delete the parts which mentions it. 
I made some test using the given system and by using the "ModDivider" part it works fine without mods but it doesn't with mods. 
However it does work with both if we just delete it, the changes I suggested should be the correct way the score is calculated in osu!mania now, or we could just put every value under the "ModDivider" to "1" which is a nonsense to me.
I saw that the English and French version of the new Score page were already merged on a different PR so I'll update them later.

**Comment by TicClick** [comment]
are your suggestions related to https://github.com/ppy/osu-wiki/issues/2025? I'd rather have them in a separate PR so that move and change aren't mixed together

**Comment by Satsukel** [comment]
Seems like it, I agree that we should not mix everything together to avoid confusion.
Will you make the PR for these changes or will I need to make it?

**Comment by TicClick** [comment]
I'll take care of it (there are also other things to proofread)

**Comment by Satsukel** [comment]
Alright, I'll try to keep an eye on it then, thank you.

**Comment by Walavouchey** [comment]
Is the intention here that once all information that should be split out has been removed from `Game_mode` (there are only deletions in that article) that it is then rewritten to summarise and/or link to the new places?

**Comment by Walavouchey** [comment]
It might make more sense to hold off on this change and delete/summarise stuff in `Game_mode` once all articles are in place, since with this pr you'd leave `Game_mode` in a state of containing partial information on very related stuff (having grade, accuracy & judgement sections but not score)

I roughly agree with the general direction though, see [discussion on discord](https://discord.com/channels/188630481301012481/218677502141399041/915716825373892618)

**Comment by TicClick** [comment]
should be ready for review now

File: wiki/Score/osu!/fr.md [hunk 1]
```diff
@@ -6,18 +6,7 @@ outdated_since: c78e8f94260067c49d36a55deaaf7f40cb796b39
...
-| ![](/wiki/Skinning/Interface/img/hit50.png "50") | 50 | Score obtenu lorsqu'un cercle est frappé avec du délai, mais assez faible que pour ne pas causer de Miss, lorsqu'un Slider est complété avec beaucoup de ticks ratés, ou lorsqu'un Spinner est complété avec sa jauge modérément remplie. Un score de 50 apparaît avec une couleur orange par défaut. Obtenir un 50 dans un combo empêche l'obtention d'un Katu ou d'un Geki à la fin de celui-ci. |
-| ![](/wiki/Skinning/Interface/img/hit0.png "Miss") | Miss | Score obtenu lorsqu'un cercle n'est pas frappé, ou frappé avec trop de délai, lorsqu'un Slider n'est pas frappé ou maintenu sur au moins un tick, ou lorsqu'un Spinner est complété avec sa jauge peu remplie. Obtenir un Miss réinitialise le combo actuel, et empêche l'obtention d'un Katu ou d'un Geki à la fin de celui-ci. |
-
-## Score de l'objet
+*See also: [osu! judgement system](/wiki/Gameplay/Judgement/osu!)*
```
Comment (Walavouchey) [inline wiki/Score/osu!/fr.md:9]: translations for the hatnotes can be copy-pasted from other article translations

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+no_native_review: true
+needs_cleanup: true  # see the original article
+---
+
+# osu! judgement system
```
Comment (Walavouchey) [inline wiki/Gameplay/Judgement/osu!/fr.md:6]: if english headers are going in as-is for now then we should slap `no_native_review` on all applicable translations

**Review by Walavouchey — COMMENTED** [review]
this is pretty good as a move operation, but perhaps `Combo_(score_multiplier)` should be put in `Gameplay` while at it, assuming that all articles in `Glossary` are only temporarily there anyway

File: wiki/Score/osu!/fr.md (hunk 1 above)
Comment (TicClick) [reply wiki/Score/osu!/fr.md:9]: it's unnecessary because titles will have to be changed, too

File: wiki/Gameplay/Judgement/osu!/fr.md (hunk 2 above)
Comment (TicClick) [reply wiki/Gameplay/Judgement/osu!/fr.md:6]: just `outdated` is enough imo

File: wiki/Score/osu!/fr.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Score/osu!/fr.md:9]: it's less weird to keep the english title than to keep "see also" in english

File: wiki/Gameplay/Judgement/osu!/fr.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/Gameplay/Judgement/osu!/fr.md:6]: perhaps

**Review by Walavouchey — APPROVED** [review]
@Joehuu waiting room

**Review by Joehuu — APPROVED** [review]

**Pull request merged**